src/components/TeamMember/index.css

```diff
@@ -0,0 +1,16 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.teamMemberImg {
+  border-radius: 50%;
+  width: 50%;
+  margin-bottom: 1.5em;
+  margin-top: 2em;
+}
+.memberCol {
+  text-align: center;
+  font-family: "Acme";
+  color: #590995;
+}
+.memTitle {
```

Comment:
What is memTitle, can you give more meaningful name

Reply:
I replaced it with memberTitle